src/Propel/Runtime/ActiveQuery/Criterion/AbstractCriterion.php

```diff
@@ -321,11 +321,12 @@ public function equals($obj)
         /** @var AbstractCriterion $crit */
         $crit = $obj;
 
-        $isEquiv = (((null === $this->table && null === $crit->getTable())
+        $isEquiv = (
+            ((null === $this->table && null === $crit->getTable())
```

Comment:
Do we keep yoda style for now?

Reply:
jep, code sniffer will take care here later next week